### PR TITLE
Stage damage compute

### PR DESCRIPTION
**Pull request by codymccoy**
This is Task 4

### CODE REVIEW

**Review by RyanM-RMA — CHANGES_REQUESTED** [review]
Minor changes requested

File: View/AggregatedStageDamage/AggregatedStageDamage.xaml [hunk 1]
```diff
@@ -38,18 +40,15 @@
         </Grid>
 
         <StackPanel Grid.Row="2" Margin="5" Orientation="Horizontal">
-            <RadioButton x:Name="manual_rad" Content="Manual"  Checked="manual_rad_Checked" IsChecked="{Binding IsManualRadioSelected}" Margin="5,0,0,0"/>
-            <RadioButton x:Name="calculated_rad" Content="Computed" Checked="calculated_rad_Checked" Margin="5,0,0,0"/>
+            <RadioButton x:Name="manual_rad" Content="Manual" IsChecked="{Binding IsManualRadioSelected}" Margin="5,0,0,0"/>
+            <RadioButton x:Name="calculated_rad" Content="Computed" IsChecked="{Binding IsManualRadioSelected,Converter={StaticResource InverseBooleanConverter}}" Margin="5,0,0,0"/>
```
[the remark this answers is not in the document]
Comment (codymccoy) [reply View/AggregatedStageDamage/AggregatedStageDamage.xaml:44]: removed

File: View/AggregatedStageDamage/CalculatedStageDamageControl.xaml [hunk 2]
```diff
@@ -52,17 +52,15 @@
                     </GridView>
                 </ListView.View>
             </ListView>
-           
         </GroupBox>
 
-        <Grid x:Name="editorGrid" Grid.Row="1" Grid.ColumnSpan="2" Visibility="Hidden">
+        <Grid x:Name="editorGrid" Grid.Row="1" Grid.ColumnSpan="2" Visibility="{Binding ShowChart, Converter={StaticResource BoolToVisibilityConverter}}">
```
[the remark this answers is not in the document]
Comment (codymccoy) [reply View/AggregatedStageDamage/CalculatedStageDamageControl.xaml:57]: I am using "editorGrid" but i got rid of all the other named items in the xaml.

File: View/AggregatedStageDamage/CalculatedStageDamageControl.xaml [hunk 3]
```diff
@@ -40,8 +41,7 @@
 
             <Button x:Name="calculate_btn" Grid.Row="3" Content="Compute Curves" Grid.ColumnSpan="2" Width="100" HorizontalAlignment="Left" Margin="5" Click="calculate_btn_Click"/>
```
[the remark this answers is not in the document]
Comment (codymccoy) [reply View/AggregatedStageDamage/CalculatedStageDamageControl.xaml:42]: removed

File: View/AggregatedStageDamage/ManualStageDamageControl.xaml.cs [hunk 4]
```diff
@@ -32,57 +21,61 @@ public ManualStageDamageControl()
...
+                linkChartViewModel();
+            }
         }
 
         private void remove_btn_Click(object sender, RoutedEventArgs e)
         {
-            ManualStageDamageVM vm = (ManualStageDamageVM)this.DataContext;
-            vm.Remove();
+            if (DataContext is ManualStageDamageVM vm)
+            {
```
[the remark this answers is not in the document]
Comment (codymccoy) [reply View/AggregatedStageDamage/ManualStageDamageControl.xaml.cs:43]: probably. I added it.

**Review by RyanM-RMA — APPROVED** [review]
Looks good

**Pull request merged**